### ISSUE TITLE
Allow public get for view.emptyState.state

### ISSUE THREAD

**Issue by gustavodiel** [alberdev/EmptyStateKit#10]
It'd be cool to get the current state of each view in so that if it is already displaying the desired state, we don't need to replay the animation, like
```swift
func onReaload() {
    ...
    if view.emptyState.state != MyStates.noInternet {
       view.emptyState.show(MyState.noInternet)
    }
}
```

or maybe have another attribute for that, like `view.emptyState.currentState`.

If y'all agree, I could make the PR no problem!

**Comment by zzzzzyijie** [comment]
yes, i have same problem ，i need to change the style in dark mode,
like 
```
 view.emptyState.reload()

```

if support dark mode will be great
